doc/versions.json

```diff
@@ -22,8 +23,17 @@
   ],
   "COMPONENTS_BY_VERSION": {
     "latest": {
-      "ncs": "1.7.99",
-      "nrfxlib": "1.7.99",
+      "ncs": "1.8.0",
+      "nrfxlib": "1.8.0",
+      "zephyr": "2.6.99",
+      "mcuboot": "1.7.99",
+      "nrfx": "2.5",
+      "tfm": "1.3.99",
+      "matter": ""
+    },
+    "1.8.0": {
+      "ncs": "1.8.0",
+      "nrfxlib": "1.8.0",
       "zephyr": "2.6.99",
```

Comment:
I believe this should be a `2.7.0` as we are basing this NCS release on Zephyr 2.7.0 LTS.

Reply:
These now match what we list on https://projecttools.nordicsemi.no/confluence/display/NCS/nRF+Connect+SDK+1.8.0